java/src/jmri/jmrix/sprog/SprogConstants.java

```diff
@@ -25,16 +25,6 @@
 
     /* The following parameters may be overridden by scripts if the user desires */
     /**
-     * Maximum number of slots for soft command station 
```

Comment:
No need to fix now, but it seems that this change will give two lines with `/**`. I don't know if this could cause any problem with Javadoc later, and I'm happy to merge this PR as is, but I think it would be good to fix in the next PR.

```
/* The following parameters may be overridden by scripts if the user desires */
/**
/**
 * Threshold to warn of long delays between DCC packets to the rails.
```